lib/src/main/kotlin/com/lemonappdev/konsist/api/declaration/KoDeclaration.kt

```diff
@@ -15,7 +15,7 @@ interface KoDeclaration : KoNamedDeclaration {
     /**
      * Package name of the declaration.
      */
```

Comment:
I changed logic of the declaration package - but I still not sure if it makes sense. We will discuss it.